scripts/scripts.js

```diff
@@ -747,6 +747,12 @@ document.addEventListener('click', () => sampleRUM('click'));
 
 loadPage(document);
 
+function buildAdBlock(main) {
+  const section = document.createElement('div');
+  section.append(buildBlock('ads', ''));
+  main.append(section);
+}
+
```

Comment:
i think building a section allows the block loader to find the ads at the same time as all the other content, which looks like it's introducing a TON of TBT. ads will need to be delayed another way.